templates/rspec_api_documentation/api_blueprint_index.mustache

```diff
@@ -16,34 +19,49 @@ FORMAT: 1A
 ## {{ route_name }} [{{ route }}]
 {{# description }}
 
-description: {{ description }}
+{{ description }}
 {{/ description }}
 {{# explanation }}
 
-explanation: {{ explanation }}
+{{{ explanation }}}
 {{/ explanation }}
 {{# has_parameters? }}
 
 + Parameters
+
 {{# parameters }}
-  + {{ name }}{{# example }}: {{ example }}{{/ example }}{{# properties_description }} ({{ properties_description }}){{/ properties_description }}{{# description }} - {{ description }}{{/ description }}
+    + {{ name }}{{# properties_description }} ({{ properties_description }}){{/ properties_description }}{{# description }} - {{ description }}{{/ description }}
 {{/ parameters }}
 {{/ has_parameters? }}
+{{# http_methods }}
+
+{{# examples }}
+{{# requests }}
+
+### {{ action_name }} [{{ http_method }}]
+{{# explanation }}
+
+{{{ explanation }}}
+{{/ explanation }}
 {{# has_attributes? }}
 
-+ Attributes (object)
++ Parameters
+
 {{# attributes }}
-  + {{ name }}{{# example }}: {{ example }}{{/ example }}{{# properties_description }} ({{ properties_description }}){{/ properties_description }}{{# description }} - {{ description }}{{/ description }}
```

Comment:
`example` is the RSpec example in this case, which we don't want. I wanted to render the actual value of the param here, but I don't think it's possible without some serious refactoring. I did add the ability to define the "default" though.

Anyhow, if you have an idea how we could get the actual parameter value in here that would be amazing 🙏